src/language-html/utils.js

```diff
@@ -488,12 +488,28 @@ function getNodeCssStyleDisplay(node, options) {
     }
   }
 
+  const isUnknownNamespaceElement =
+    node.type === "element" &&
+    node.namespace &&
+    !["html", "svg"].includes(node.namespace);
+
   switch (options.htmlWhitespaceSensitivity) {
     case "strict":
       return "inline";
     case "ignore":
       return "block";
     default:
+      if (node.type === "element" && isUnknownNamespaceElement) {
+        const display = CSS_DISPLAY_TAGS[node.name];
+        if (display) {
+          return display;
+        }
+
+        if (!HTML_TAGS[node.name]) {
+          return "block";
```

Comment:
This should be `inline` (`CSS_DISPLAY_DEFAULT`). The formatting of unknown (custom) tags should not depend on whether those tags are nested in a tag whose name contains a colon.

Reply:
If set to inline.

Will format to 
```html
<foo:bar>
  <div>block block block</div>
  <div>block block block</div>
  <div>block block block</div>
  <span> inline inline </span><span> inline inline </span
  ><span> inline inline </span><span> inline inline </span
  ><html:button> known inline</html:button></foo:bar
>
```

because `<foo:bar>` is inline, seems wired.

Reply:
`<foo-bar>` is treated the same way, the safest way to deal with unknown elements. There should be no difference between `foo-bar` and `foo:bar`.